Assignment2/claubaldman-ClaudioBaldman.ttl

```diff
@@ -0,0 +1,22 @@
+@base <http://www.oeg-upm.net/ontologies/class/>
+@prefix class: <http://www.oeg-upm.net/ontologies/class#>
+@prefix sensor=<http://www.oeg-upm.net/ontologies/sensor#>
+@prefix computer=<http://www.oeg-upm.net/ontologies/computer#>
+@prefix measurement=<http://www.oeg-upm.net/ontologies/measurement#>
+@prefix user=<http://www.oeg-upm.net/ontologies/user#>
+   
+    :Class01
+                class: includes:  Sensor029

```

Comment:
missing a final . for all lines where a triple finishes. Same with ; when it makes sense